bdd/spec/007_comparator_spec_lnn.sh

```diff
@@ -0,0 +1,68 @@
+Include build_tools.sh
+
+Describe "Comparators"
+  Describe "Equals"
+    before() {
+      lnn_sourceToAll "
+        from @std/app import start, stdout, exit
+
+        on start {
+          const i8val = 0;
```

Comment:
Does this mean it tries to default to the smallest matching type instead of the largest? Or should this be fixed to `const i8val: int8 = 0;`?

Reply:
no - it's a test on the constraint from `emit exit i8val` at the bottom

Reply:
Ahh, got it. Can we add a comment saying that?